Puppetfile.tracking

```diff
@@ -21,7 +21,7 @@ mod 'simp-rsync_data',
 
 mod 'simp-adapter',
   :git => 'https://github.com/simp/simp-adapter',
-  :branch => 'master'
+  :branch => '0.1.0'
```

Comment:
rake puppetfile:check_module[simp-adapter]
rake aborted!
Could not find a tagged version of 'simp-adapter' in 'Puppetfile.tracking'